sound/hybrid_font.h

```diff
@@ -799,8 +802,58 @@ class HybridFont : public SaberBase, public Looper {
 	SaberBase::DoEffect(EFFECT_POSTOFF, saved_location_);
       }
     }
+    // Wait for retraction sounds to finish, then play idle.wav
+#ifdef ENABLE_IDLE_SOUND
+    if (idling_) {
+      if (GetWavPlayerPlaying(&SFX_in) || GetWavPlayerPlaying(&SFX_pstoff)) {
+        return;
+      } else {
+        idling_ = false;
+        StartIdleSound();
+      }
+    }
+#endif
   }
-  bool swinging_ = false;
+  void CheckStopIdle() {
+#ifdef ENABLE_IDLE_SOUND
+    // Since idle sound re-uses hum_player,this will prevent
```

Comment:
This comment doesn't make sense to me.
Swings will trigger a hum regardless of whether we use the hum_player or not in a monophonic font.

Reply:
This came about after changing a dedicated idle_player_ to hum_player_ instead.
HOWEVER. I just tested again now and everything works fine without CheckStopIdle() even existing...?

Unless I'm crazy because hum was indeed playing unexpectedly during development for some reason and I thought this was needed.
A swing command would gaplessly transition to the hum sound once the swing/slash/spin sound was finished..
The comment was to explain the guarding condition, which was really only specifically for the off chance that a serial command is sent while the blade is off anyway.

Actually, if CheckStopIdle() does exist, it stops idle after the swing command (obviously), which I guess could be considered a negative behavior anyway.
So... I've just removed it.
Oh well.